src/triton_cli/templates/trt_llm/tensorrt_llm_bls/config.pbtxt

```diff
@@ -24,6 +24,7 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+name: "tensorrt_llm_bls"
```

Comment:
remove the `name` field from model configs - it will break if model folder is renamed and is unnecessary.

Reply:
alternatively you can update the trtllm config parsing logic to replace this line if found if you'd like.

Reply:
Removed the `name` field from `tensorrt_llm_bls` model config template.